### PR TITLE
chore: formatting readme and removing gha and renovate

### PR DESCRIPTION
**Pull request by esacteksab**
- format readme
- delete gha
- delete renovate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # DEPRECIATION NOTICE
```
Comment (rpdelaney) [inline README.md]:
```suggestion
# DEPRECATION NOTICE
```

**Review by rpdelaney — CHANGES_REQUESTED** [review]
Also, do we want to be removing GHA?

**Comment by esacteksab** [comment]
> Also, do we want to be removing GHA?

Yes. No sense in running this on a repo that will be archived.

**Review by rpdelaney — APPROVED** [review]
Ok. My $0.02 is that GHA doesn't run on archived repos, and in the unlikely event that we do un-archive this, we'll have to revert the removal.

**Pull request merged**